homeassistant/components/samsungtv/media_player.py

```diff
@@ -103,7 +106,26 @@ def __init__(self, config_entry, on_script):
             "port": config_entry.data.get(CONF_PORT),
             "host": config_entry.data[CONF_HOST],
             "timeout": 1,
+            "token_file": None,
         }
+        if config_entry.data[CONF_PORT] == 8002:
+            self._config["token_file"] = self._get_token_file()
+
+    def _get_token_file(self):
```

Comment:
This is already done in the config flow.

Save the token path in the config entry.

Reply:
Done